app/models/core/email.rb

```diff
@@ -2,7 +2,7 @@
 #
 # Table name: core_emails
 #
-#  id         :uuid             not null, primary key
+#  id         :bigint(8)        not null, primary key
```

Comment:
@RichardLaskowski This concerns me. Now sure why the core_emails has an integer `id`. The migration seems corrected and unchanged. On this branch can you drop your db, and recreate and migrate? Then annotate again and let's see if this gets fixed. You can see the issue in the db/schema.rb file too